www/CameraPreview.js

```diff
@@ -89,24 +88,35 @@ CameraPreview.getSupportedPictureSizes = function(onSuccess, onError){
 };
 
 CameraPreview.setFlashMode = function(flashMode, onSuccess, onError) {
-  flashMode = flashMode.toLowerCase();
-  if(flashMode === 'off'){
-    flashMode = 0;
-  }else if(flashMode === 'on'){
-    flashMode = 1;
-  }else if(flashMode === 'auto'){
-    flashMode = 2;
-  }else if(flashMode === 'torch' && navigator.userAgent.match(/Android/i)  == "Android"){
-    flashMode = 3;
-  }else{
-    return false;
-  }
-
   exec(onSuccess, onError, PLUGIN_NAME, "setFlashMode", [flashMode]);
 };
 
 CameraPreview.tapToFocus = function(xPoint, yPoint, onSuccess, onError){
   exec(onSuccess, onError, PLUGIN_NAME, "tapToFocus", [xPoint, yPoint]);
 };
 
+CameraPreview.FlashMode = {
```

Comment:
I would prefer to keep it this way, as people probably read the doc of the cordova camera plugin as well.

Do you want me to capitalize the names? I guess it would make it look more like constants.